src/lib/utils/gql/queries.js

```diff
@@ -15,6 +15,7 @@ export const queryEvents = gql`
 				picture {
 					id
 					url
+					small: url(transformation: { image: { resize: { width: 200, height: 200, fit: clip } } })
```

Comment:
This can be anything and changed if small doesn't fit, there can be many sizes added here with  GQL labels